### DISCUSSION TITLE
Geography Property Linked in another table

### DISCUSSION THREAD

**Discussion by deaquino** [JasperFx/marten#2436] [General]
I have a model that includes a Property that is of type NetTopologySuite Geometry/Geography and I want to store it in a separate table or column and not in the jsonb one.

Can it be done? How?

I doesn't have to be a Geography property, just a property that you don't want to serialize a jsonb and a specific column is required to store them.

Cheers!

Jaime

**Comment by mysticmind** [comment]
Marten always uses JSONB for storing the serialized data. You could mark the property with `[JsonIgnore]` to exclude it from being serialized and stored. With regards to storing it separately, you can create a separate table outside of Marten and use https://martendb.io/documents/execute-custom-sql.html#execute-custom-sql-in-session to insert data using the same session or get the connection to run a custom sql via Npgsql.
